src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -85,7 +85,11 @@ public static void MapRequestScrollTo(IScrollViewHandler handler, IScrollView sc
 		{
 			if (args is ScrollToRequest request)
 			{
-				handler.PlatformView.SetContentOffset(new CoreGraphics.CGPoint(request.HorizontalOffset, request.VerticalOffset), !request.Instant);
+				var availableScrollHeight = handler.PlatformView.ContentSize.Height - handler.PlatformView.Frame.Height;
```

Comment:
We could avoid hitting the handler.PlatformView path so many times.
`var scrollView = handler.PlatformView;`, `var contentSize= scrollView.ContentSize;`, etc.

Reply:
+1 on this one. I know we are in the a single thread env (main) but it is also a good practice in general to make sure the is not changed.

Reply:
done